src/main/java/com/revature/security/CustomAuthenticationFilter.java

```diff
@@ -129,10 +129,11 @@ public String get_SHA_512_SecureHash(String passwordToHash, String salt) {
 	 * @param header The retrieved header from the request object
 	 */
 	public boolean validateHeader(String header) {
-		if (header == null) {
-			return false;
-		}
-		return header.equals(get_SHA_512_SecureHash(zuulConfig.getSecret(), zuulConfig.getSalt()));
+//		if (header == null) {
```

Comment:
Uncomment code for local work around and repush.

Reply:
Fixed.